src/scalarstats.jl

```diff
@@ -509,7 +509,8 @@ crossentropy(p::AbstractArray{T}, q::AbstractArray{T}, b::Real) where {T<:Real}
 """
     kldivergence(p, q, [b])
 
-Compute the Kullback-Leibler divergence of `q` from `p`, optionally specifying
+Compute the Kullback-Leibler divergence from `q` to `p`,
+or the sum `pi * log(pi / qi)`, optionally specifying
```

Comment:
```suggestion
i.e. the sum ``p_i * log(p_i / q_i)``, optionally specifying
```

While you're at it, maybe mention that it's also called relative entropy?